pkg/operators/v1alpha1/clusterserviceversion_types.go

```diff
@@ -182,7 +183,7 @@ type WebhookDescription struct {
 // GetValidatingWebhook returns a ValidatingWebhook generated from the WebhookDescription
 func (w *WebhookDescription) GetValidatingWebhook(namespace string, namespaceSelector *metav1.LabelSelector, caBundle []byte) admissionregistrationv1.ValidatingWebhook {
 	return admissionregistrationv1.ValidatingWebhook{
-		Name:                    w.Name,
+		Name:                    w.GenerateName,
```

Comment:
Is this potentially problematic?

Reply:
Shouldn't be given that:
- OLM ensures that the GenerateName is unique
- The fact that ValidatingWebhook is not a cluster resource, ValidatingWebhookConfiguration is.